www/convertUtils.js

```diff
@@ -28,10 +28,19 @@ module.exports = {
     */
     toCordovaFormat: function (contact) {
         var value = contact.birthday;
-        try {
-          contact.birthday = new Date(parseFloat(value));
-        } catch (exception){
-          console.log("Cordova Contact toCordovaFormat error: exception creating date.");
+        if (value !== null) {
```

Comment:
Why the null check ? If null was returned from the native side, this case will be handled by the ```if (isNaN(contact.birthday.getTime()))``` check you've got below